manager-bundle/src/Command/MaintenanceModeCommand.php

```diff
@@ -77,6 +77,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
     private function enable(string $templateName, string $templateVars): void
     {
+        // Change the error template namespace (see #8195)
+        if (str_starts_with($templateName, '@ContaoCore/Error') && !$this->twig->getLoader()->exists($templateName)) {
+            $templateName = str_replace('@ContaoCore/Error', '@Contao/error', $templateName);
```

Comment:
Does this need to be dynamic? I would use the hardcoded template name:

```suggestion
        if ($templateName === '@ContaoCore/Error/service_unavailable.html.twig' && !$this->twig->getLoader()->exists($templateName)) {
            $templateName = '@Contao/error/service_unavailable.html.twig';
```

Alternatively, I would at least add a slash:
```suggestion
        if (str_starts_with($templateName, '@ContaoCore/Error/') && !$this->twig->getLoader()->exists($templateName)) {
            $templateName = str_replace('@ContaoCore/Error/', '@Contao/error', $templateName);
```

Reply:
>Does this need to be dynamic? I would use the hardcoded template name:

Had it like that in the first place in case you would use the maintenance mode template with the --template prefix in your systems since people may have had their own commands built but could revert it. Wdyt would be best here @m-vo ?

Reply:
If you had specified a custom template using `--template`, this would not have been affected by the changes anyway. You are free to use whatever namespace you like and it should not be altered.

Reply:
Guessind peoples templates would not have changed so it's 1, yeah.